### PR TITLE
Details for window.maximize()

### PR DESCRIPTION
**Pull request by talha-ah**
> Issue number that this PR fixes (if any). For example: 'Fixes #987654321'
Not reported yet

> What was wrong/why is this fix needed? (quick summary only)
Content was needed for window.maximize()

> Anything else that could help us review it

### CODE REVIEW

**Review by wbamberg — COMMENTED** [review]
Thanks for this PR! But I think we should:

* delete the entry here for `maximize()`
* delete the entry below it for `minimize()`
* delete the linked page for `minimize()` (https://github.com/mdn/content#deleting-a-document)

These functions are apparently XUL-only, which means they are proprietary Firefox functions, which we should not document here.

If you'd like to make these changes, please go ahead :).

File: files/en-us/web/api/window/index.md [hunk 1]
```diff
@@ -232,8 +232,8 @@ _This interface inherits methods from the {{domxref("EventTarget")}} interface a
   - : Returns the selection object representing the selected item(s).
 - {{domxref("Window.matchMedia()")}}
   - : Returns a {{domxref("MediaQueryList")}} object representing the specified media query string.
-- {{domxref("Window.maximize()")}}
-  - : {{todo("NeedsContents")}}
+- {{domxref("Window.maximize()")}} (top-level XUL windows only)
+  - : Miximizes the window.
```
Comment (teoli2003) [inline files/en-us/web/api/window/index.md:236]:
```suggestion
  - : Maximizes the window.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
There is a typo to fix.

**Comment by wbamberg** [comment]
@teoli2003 , what do you think about https://github.com/mdn/content/pull/9147#pullrequestreview-760400502 ?

**Comment by teoli2003** [comment]
Oh! Good catch! I missed that it was XUL-only. Yes, of course, like with minimize() this should be delete.

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**